dap-php.el

```diff
@@ -40,7 +40,7 @@
   (-> conf
       (dap--put-if-absent :dap-server-path dap-php-debug-program)
       (dap--put-if-absent :type "node")
-      (dap--put-if-absent :cwd projectile-project-root)
```

Comment:
I think that it should be `(lsp-find-session-folder (lsp-session) (buffer-file-name))` sorry for the unclear comment.

Reply:
I could have tested it... Did it this time; last commit what you want?